README.md

```diff
@@ -100,17 +102,19 @@ To enable Selectrum, simply add to your init-file:
 (selectrum-mode +1)
 ```
 
-Now all of your favorite Emacs commands will automatically use
-Selectrum. However, the default sorting and filtering is very basic.
-The standard filtering function uses `completion-styles`. You can make
-use of the built-in `substring` and `flex` style to enhance these a
-bit but it is recommended to use packages which improve this further.
-There is [`prescient.el`](https://github.com/raxod502/prescient.el)
-which provides functions for more intelligent sorting and filtering
-and [`orderless`](https://github.com/oantolin/orderless) which
-provides an enhanced and flexible style for `completion-styles`. To
-setup `prescient` simply install `selectrum-prescient` package from
-MELPA and add the following to your init-file:
+Now all completion commands will automatically use Selectrum. The
+default filtering function uses `completion-styles` which only provide
+a few simple filtering methods. You can additionally make use of the
+built-in `substring` and `flex` style which are more advanced but it
+is recommended to use packages which improve this further.
+
+Generally Selectrum doesn't integrate features you might be used to
+from bigger completion frameworks. Instead the focus is on providing
+an enhanced completion UI and compose with other packages which also
+stay within the constraints of the standard Emacs API. If you mainly
```

Comment:
```suggestion
stay within the constraints of the standard Emacs API. 

If you mainly
```

Obviously you'd need to adjust the fill if you do this, but basic idea is to split paragraphs on key points so they stand out more.